pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.13.5
```

Comment:
- please use 0.13.6-dev
- please run `pub run build_runner build` (to update the version.dart file)